### PR TITLE
Feature/main page part 1

### PR DESCRIPTION
**Pull request by ari1111i**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import { MainPage } from './components/MainPage'
+
 export function App() {
-   return <div className="App">App</div>
```
Comment (NuraiymMamatova) [inline src/App.js]: Please clean App.jsx: replace <MainPage/> to App

File: src/components/MainPage.jsx [hunk 2]
```diff
@@ -0,0 +1,240 @@
+import React, { useState, useEffect } from 'react'
+
+import { styled } from '@mui/material'
+
+import { MainContent } from './MainContent'
+
+import img from '../assets/images/main-like.png'
```
Comment (NuraiymMamatova) [inline src/components/MainPage.jsx]: Import image like component from Index.js in assets package

File: src/components/MainPage.jsx [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+
+import images from '../assets/images/main-dis.png'
+
+import image from '../assets/images/main-loy.png'
+
+const DivContainer = styled('div')`
```
Comment (NuraiymMamatova) [inline src/components/MainPage.jsx]: If the elements have similar styles, you can use only one container for this, for example:  const DivContainer = styled('div')`
   display: flex;
`

File: src/components/MainPage.jsx [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+`
+const Paragrap = styled('p')`
+   margin-left: 11.875rem;
+   font-size: 1.063rem;
+   margin-top: 0.625rem;
+`
+const Par = styled('p')`
+   color: #8639b5;
+   font-size: 2.5rem;
+   font-family: Arial, Helvetica, sans-serif;
```
Comment (NuraiymMamatova) [inline src/components/MainPage.jsx]: we don't need to use these fonts. font styles already exist in index.css

File: src/components/MainPage.jsx [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+            </div>
+            <div>
+               <Par>{checkCount} K+</Par>
+               <Paragraph>Реализованный благотворительной помощи</Paragraph>
+            </div>
+         </Div>
+         <DivBlock>
+            <DivContainer>
+               <div>
+                  <Img src={img} alt="" />
```
Comment (NuraiymMamatova) [inline src/components/MainPage.jsx]: could you write anything to alt attribute

**Review by NuraiymMamatova — APPROVED** [review]

**Review by Satybaldyeva — APPROVED** [review]

File: src/components/MainContent.jsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+   color: white;
+   margin-left: 56.25rem;
+   font-size: 2.188rem;
+   padding-top: 9.375rem;
+`
+const Ps = styled('P')`
+   color: white;
+   padding-left: 56.25rem;
+   margin-top: 1.875rem;
+   margin-right: 6.25rem;
```
Comment (AvazMAX) [inline src/components/MainContent.jsx]: мындай padding тер margin минустар болбойт margin коп болсо дагы болбойт flexbox колдонушунар керек

**Comment by AvazMAX** [comment]
image
ul li кылып коесунбу)

**Pull request merged**